### PR TITLE
[master] Request requires a second parameter

### PR DESCRIPTION
**Pull request by DeepDiver1975**
fixes self

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@MorrisJobke @th3fallen @LukasReschke please review - THX

File: tests/lib/appframework/controller/OCSControllerTest.php [hunk 1]
```diff
@@ -36,12 +36,13 @@ class OCSControllerTest extends \Test\TestCase {
 
 	public function testCors() {
 		$request = new Request(
-			array('server' => array('HTTP_ORIGIN' => 'test'))
+			array('server' => array('HTTP_ORIGIN' => 'test')),
+            $this->getMock('\OCP\Security\ISecureRandom')
```
Comment (MorrisJobke) [inline tests/lib/appframework/controller/OCSControllerTest.php]: intendation

**Comment by MorrisJobke** [comment]
Beside my comment :+1:

**Comment by LukasReschke** [comment]
:+1:

**Pull request merged**